ide/extbrowser/src/org/netbeans/modules/extbrowser/NbDdeBrowserImpl.java

```diff
@@ -212,7 +380,16 @@ private String realDDEServer () {
         }
         
         try {
-            String cmd = getDefaultOpenCommand ();
+            String cmd = NbDdeBrowserImpl.getDefaultWindowsOpenCommand();
+            
+            /** if not found with getDefaultWindowsOpenCommand function
+             *  fallback to previous method
+             */
+            if (cmd.isEmpty())
+                {
+                    cmd = getDefaultOpenCommand();
+                }
```

Comment:
I left that there since I'm not sure whether the registry location I'm querying is valid in previous versions of windows that are still supported by netbeans